.php-cs-fixer.dist.php

```diff
@@ -28,7 +28,6 @@
     ->exclude(['dev-tools/phpstan', 'tests/Fixtures'])
     ->in(__DIR__)
     ->append([
-        __DIR__.'/dev-tools/doc.php',
```

Comment:
I believe we can remove whole `append()` call because it does not make sense to keep only comment there.

Reply:
Done.